src/utils/domain.ts

```diff
@@ -102,13 +75,8 @@ export const getInitialFromName = (name: string) => {
   // Returns PR if name is "Peter Edwards Roman"
   // Returns PR if name is "Preben"
 };
+export const getAvatarPathWithFallback = (avatar?: string, name?: string) => {
```

Comment:
This function needs to return a `string` but it's returning `string | undefined`. [`useImageUploader()`](https://github.com/coordinape/coordinape/pull/1574/files#diff-8fe7cd1d94726f02888d91e0873c8e24ba50cd7daf274077e798cfed738fa4a2R28) expects a `string`, but it can potentially get an `undefined` and that's why you're getting

> Type 'undefined' is not assignable to type 'string'.ts(2345)